formats/developers/ome-units.txt

```diff
@@ -0,0 +1,253 @@
+The OME System of Units
+=======================
+
+.. topic:: Overview
+
+   This sections explains the OME system of Units.
```

Comment:
This repeats the section heading and is redundant.  Could it be removed or modified to add something additional to the title?